components/dashboard/src/teams/TeamUsageBasedBilling.tsx

```diff
@@ -253,17 +255,22 @@ function UpdateLimitModal(props: {
                         min={0}
                         value={newLimit}
                         className="rounded-md w-full truncate overflow-x-scroll pr-8"
-                        onChange={(e) => setNewLimit(parseInt(e.target.value || "1", 10))}
+                        onChange={(e) => setNewLimit(e.target.value)}
                     />
                 </div>
             </div>
             <div className="flex justify-end mt-6 space-x-2">
                 <button
                     className="secondary"
                     onClick={() => {
-                        if (typeof newLimit === "number") {
-                            props.onUpdate(newLimit);
+                        if (!newLimit) {
+                            return;
+                        }
+                        const n = parseInt(newLimit, 10);
+                        if (typeof n !== "number") {
+                            return;
```

Comment:
Here we silently drop errors here, and actually happened to me: Had a hidden space in front :see_no_evil: . Atm we just keep the modal (no feedback)

:cloud:  We could also have an `<Alert />`, and shows parsing errors there. But I guess that's sth for a quick follow-up.

Reply:
Agreed, some validation would be nice. We could also trim the input.

Also, I don't know whether we should allow setting `0` (currently reverts the limit to `undefined` it seems?) and negative values should probably not work.